modules/patient-portal/src/main/frontend/src/patient-portal/SurveyInstructionsConfiguration.jsx

```diff
@@ -78,7 +78,7 @@ function SurveyInstructionsConfiguration() {
         onConfigSaved={() => setHasChanges(false)}
         >
           <List className={classes.formEntries}>
-            { Object.keys(labels).map(category => { return (<>
+            { Object.keys(labels).map(category => { return (<span key={category + "Wrapper"}>
```

Comment:
By adding this span, we're wrapping block elements in an inline element, resulting in invalid html. Instead, put the key on the `Fragment`:
```suggestion
            { Object.keys(labels).map(category => { return (<React.Fragment key={category + "Wrapper"}>
```